zerver/lib/actions.py

```diff
@@ -6958,6 +6964,7 @@ def do_resend_user_invite_email(prereg_user: PreregistrationUser) -> int:
         "prereg_id": prereg_user.id,
         "referrer_id": prereg_user.referred_by.id,
         "email_body": None,
```

Comment:
Do you know why this one has an `email_body` and the other doesn't?

I think it could be a good project to define a `zerver/lib/queue_types.py` with a set of data classes, at least one for each queue worker, to make sure that these are more of a documented interface.  Let me start a thread about that.

Reply:
I think `email_body` here was missed to be removed in 47fcb27e393.